WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Print Shipping Label/PrintShippingLabelViewController.swift

```diff
@@ -110,10 +118,14 @@ private extension PrintShippingLabelViewController {
         }
     }
 
-    func configurePrintButton() {
-        printButton.applyPrimaryButtonStyle()
-        printButton.setTitle(Localization.printButtonTitle, for: .normal)
-        printButton.on(.touchUpInside) { [weak self] _ in
+    func configureReprintButton() {
+        guard printType == .reprint else {
+            reprintButton.isHidden = true
+            return
+        }
+        reprintButton.applyPrimaryButtonStyle()
+        reprintButton.setTitle(Localization.printButtonTitle, for: .normal)
+        reprintButton.on(.touchUpInside) { [weak self] _ in
             self?.printShippingLabel()
         }
     }
```

Comment:
It's not clear to me why we have a `reprintButton`, since we already have a row for it.

Reply:
The `reprintButton` is from the original implementation of the `ReprintShippingLabelViewController`. The design for that screen has the print button at the bottom of the screen, and as I understand it, there's no way to do that with a table row.

So I decided to leave that `reprintButton` so it could be pinned to the bottom of the screen for reprinting a label, and introduced a row for the print button for printing a new label. Is there a better way to handle those differences in the design?

Reprint label (print button at bottom) | Print new label (print button in table)
-|-
![Simulator Screen Shot - iPhone 11 Pro - 2021-07-06 at 15 07 25](https://user-images.githubusercontent.com/8658164/124752468-cced0e00-df1f-11eb-9bdf-996d1cae8e1b.png)|![Simulator Screen Shot - iPhone 11 Pro - 2021-07-06 at 15 08 23](https://user-images.githubusercontent.com/8658164/124752473-ce1e3b00-df1f-11eb-90dd-d729479d35aa.png)